### PR TITLE
Changing requirements

### PR DESCRIPTION
**Pull request by random82**
Fixed some wording to emphasize being a proactive supporter of a technical direction, not only follower of whatever change customer want to proceed with.

### CODE REVIEW

File: Consulting.md [hunk 1]
```diff
@@ -58,7 +58,7 @@
 - I am good at recognising problems outside the scope of my work and eagerly get involved in improving our product, our environment and our processes.
 - I happily tackle the most difficult problems the team is facing and usually solve them with an impressive balance of finesse and pragmatism.
 - Every venture has mundane tasks but I'm the one that can be trusted to get them done, and get them done right - usually with a handful of ingenuity.
-- I adapt quickly to changing requirements and typically structure my approach to cater for this.
+- I confidently and constructively challenge changing requirements and support my customer in structuring technical approach that helps to adapt
```
Comment (kjacobsen) [inline Consulting.md]: Perhaps that should read ...support my customer in structuring a technical approach...?

**Comment by random82** [comment]
Anyone interested in merging this one?

**Comment by kjacobsen** [comment]
I am happy to merge if no one else wants to.

**Comment by andrewabest** [comment]
I don't like the change from 'adapt to change' to 'challenge change' - I think it sets the wrong precedent. I get the intent of the PR - we need to make sure customers are aware of the effect of change on delivery, expecially constant, unconstrained change. 

But I don't like removing the fact that we should foremost adapt to change, and use an approach that allows us to adapt easily - a.k.a being agile.

**Comment by kjacobsen** [comment]
We really need to be adaptable to change, whilst challenging change.

**Comment by andrewabest** [comment]
How about

> I recognize and accept that change is constant, and my approach allows me to tactfully challenge, or confidently adapt, depending on which I deem appropriate.

**Comment by danieljanczak** [comment]
My take:

> I challenge unnecessary changes, but actively support and adapt to necessary ones.

**Comment by robdmoore** [comment]
Personally, I really like @andrewabest's suggestion.

**Comment by andrewabest** [comment]
@random82 any input to the suggestions?

**Comment by random82** [comment]
@andrewabest version is in. Happy to merge

**Comment by andrewabest** [comment]
:shipit:

**Pull request merged**